blocks/cards/cards.css

```diff
@@ -30,22 +29,21 @@ main .block.cards {
   box-shadow: 0 2px 12px 0 rgb(60 70 80 / 20%);
   border: none;
   transition: all 0.15s ease-in-out;
-  width: 16.875rem;
-  padding: 12px 1rem;
   margin: 0 0 1rem;
   align-items: center;
   text-align: left;
+  flex: 1;
+  background-color: var(--white);
 }
 
 .cards > ul > li:hover {
   box-shadow: 0 4px 1rem 0 rgb(60 70 80 / 40%);
 }
 
 .cards > ul > li .date {
-    font-size: var(--body-font-size-xs);
-    color: var(--transparent-grey-color);
-    font-weight: var(--font-weight-medium);
-    padding-bottom: 0.5rem;
+  font-size: var(--body-font-size-xs);
```

Comment:
I think you'd still need to retain the color. The date is lighter in shade on the reference site.

Reply:
Its for accessibility